cmd/api/main.go

```diff
@@ -101,10 +103,16 @@ func main() {
 		mapsApiKey:  mapsApiKey,
 	}
 
+	c := cors.New(cors.Options{
+		AllowedOrigins: []string{"http://localhost:*"},
```

Comment:
Seems like this should be a config setting when you implement some sort of .env or similar environment variable storage?

Reply:
I dunno...like, I need it here to run locally. I expect the UI and API will run on the same domain when deployed, right?